cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/hbase/AbstractHBaseDataSetAdmin.java

```diff
@@ -190,7 +190,7 @@ public static void setVersion(HTableDescriptor tableDescriptor) {
 
   public static ProjectInfo.Version getVersion(HTableDescriptor tableDescriptor) {
     String value = tableDescriptor.getValue(CDAP_VERSION);
-    return value == null ? null : new ProjectInfo.Version(value);
+    return new ProjectInfo.Version(value);
```

Comment:
This was like this before. @abaranau changed it to return null except of 0 if the table does not have a version and that breaks thing so I discussed with him and we decided to revert it back.